airbyte-config/init/src/main/resources/seed/source_specs.yaml

```diff
@@ -714,6 +714,15 @@
           type: "string"
           description: "Api key of bamboo hr"
           airbyte_secret: true
+        custom_reports_fields:
+          type: "string"
+          default: ""
+          description: "Comma-separated list of fields to include in custom reports."
+        custom_reports_include_default_fields:
+          type: "boolean"
+          default: true
+          description: "If true, the custom reports endpoint will include the default\
+            \ fields defined here: https://documentation.bamboohr.com/docs/list-of-field-names."
```

Comment:
Do you know why there are spec changes in here when there is no version bump for these connector definitions? I was under the impression that the specs should only change if there is a version change for the connector